sql/core/src/test/scala/org/apache/spark/sql/execution/SQLViewTestSuite.scala

```diff
@@ -249,6 +249,15 @@ abstract class SQLViewTestSuite extends QueryTest with SQLTestUtils {
       }
     }
   }
+
+  test("SPARK-34260: replace existing view using CREATE OR REPLACE") {
+    val viewName = createView("testView", "SELECT * FROM (SELECT 1)")
```

Comment:
shall we put it inside `withView {...}`?

Reply:
actually no, because we can't get the viewName before calling `createView`. for example, after creating a global temp view, the viewName will be global_temp.testView.
with this method, we can use `withView` for temp, global, permanent view test.